kong/plugins/opentelemetry/handler.lua

```diff
@@ -150,31 +148,36 @@ function OpenTelemetryHandler:access()
   propagation_set("preserve", header_type, translate_span_trace_id(root_span), "w3c")
 end
 
-function OpenTelemetryHandler:log(conf)
-  ngx_log(ngx_DEBUG, _log_prefix, "total spans in current request: ", ngx.ctx.KONG_SPANS and #ngx.ctx.KONG_SPANS)
+local function get_params(config)
+  local key = config.__key__
+  local queue = unpack({config.queue or {}})
+  if config.batch_span_count then
+    ngx.log(ngx.WARN, string.format(
```

Comment:
localize `string.format` `ngx.log` `ngx.WARN`

Reply:
It does not matter at this point and it is easier to understand code if global variables are not localized.